Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -19,12 +19,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\Shared\DependencyTrackingTelemetryModule.cs;..\Shared\SanitizedHostList.cs;..\Shared\Implementation\DependencyCollectorEventSource.cs;..\Shared\Implementation\ApplicationInsightsUrlFilter.cs;..\Shared\Implementation\RemoteDependencyConstants.cs;..\..\Common\CorrelationIdLookupHelper.cs;..\..\Common\CrossComponentCorrelationEventSource.cs;..\..\Common\ExceptionUtilities.cs;..\..\Common\ICorrelationIdLookupHelper.cs;..\..\Common\RequestResponseHeaders.cs;..\..\Common\HeadersUtilities.cs;..\..\Web\Web.Net45\Implementation\SdkVersionUtils.cs" Exclude="bin\**;obj\**;**\*.csproj;packages\**" />
+    <Compile Include="..\Shared\Implementation\HttpCoreDiagnosticSourceListener.cs;..\Shared\Implementation\HttpHeadersUtilities.cs;..\Shared\Implementation\PropertyFetcher.cs;..\Shared\DependencyTrackingTelemetryModule.cs;..\Shared\SanitizedHostList.cs;..\Shared\Implementation\DependencyCollectorEventSource.cs;..\Shared\Implementation\ApplicationInsightsUrlFilter.cs;..\Shared\Implementation\RemoteDependencyConstants.cs;..\..\Common\CorrelationIdLookupHelper.cs;..\..\Common\CrossComponentCorrelationEventSource.cs;..\..\Common\ExceptionUtilities.cs;..\..\Common\ICorrelationIdLookupHelper.cs;..\..\Common\RequestResponseHeaders.cs;..\..\Common\HeadersUtilities.cs;..\..\Web\Web.Net45\Implementation\SdkVersionUtils.cs" Exclude="bin\**;obj\**;**\*.csproj;packages\**" />
```

Comment:
I do not want to include whole Shared folder for several reasons
1. It contains code that depend on System.Http -related code (WebRequests)
2. It adds A LOT of unused code into the lib that will just make it bigger